distributed/tensor_parallelism/sequence_parallel_example.py

```diff
@@ -1,11 +1,17 @@
-import argparse
-
+import os
 import torch
-import torch.multiprocessing as mp
+import torch.nn as nn
+import torch.nn.functional as F
+
+from torch.distributed._tensor.device_mesh import init_device_mesh
+from torch.distributed._tensor import DeviceMesh, DTensor, Replicate, Shard
+
+from torch.distributed.tensor.parallel import (
+    parallelize_module,
+    ColwiseParallel,
+    RowwiseParallel,
+)
 
-from torch.distributed._tensor import DeviceMesh
-from torch.distributed.tensor.parallel import parallelize_module
-from utils import cleanup, setup, ToyModel
 
 try:
```

Comment:
also this `try` catch import block can be removed

Reply:
removed!